### ISSUE TITLE
Feature suggestion : Why just FB?

### ISSUE THREAD

**Issue by vibhas777** [yonahforst/focusbook#5]
Can we modify this app so that the user can configure the site that he  wants the app to work on and not just fb?

**Comment by vexin217** [comment]
Kind of, you can trivially add the websites' urls to the manifest.json file. But you have to see whether the extension plays nicely with the website' elements/css etc.

**Comment by yonahforst** [comment]
it should work with any site. it would be cool to make a list of supported sites, along with their color schemes and maybe some site specific phrases.

**Comment by evilbuck** [comment]
Would this be desired as a [user] configurable option? Something in the options page perhaps? This would require a permissions change from `facebook.com/*` to all sites. Is this acceptable?

**Comment by yonahforst** [comment]
If I was doing it, I would start by adding support for specific sites. Youtube, twitter, vk, reddit.

It would be nice if the extension looked and talked differently depending on the site. I think it's important to have an integrated and contextual experience. Otherwise it's just this ugly annoying thing that always pops up.

**Comment by evilbuck** [comment]
Ahh. It sounds like it would mostly be adding sites to the manifest with some custom css for each site.

**Comment by yonahforst** [comment]
Yes. that's my gut feeling for how it should work... but i'm open to alternatives

**Comment by evilbuck** [comment]
Well, fb is my main use case. If anyone [ @vibhas777 ] has a suggestion for a particular site, perhaps some work could be done. 
On the other hand, a generic notifier/nagger could be made to work with user configured sites. The notification look is pretty generic as it is. I would just have different "themes" for specified sites and default to a light shade of grey with darker text for sites that were not specified. 
The biggest issue really being the permissions change.
